Description: adjust placement of crate dependencies and a comment
 Crates memmap2 and object are not dist-only requirements.
Bug: https://github.com/mozilla/sccache/issues/1894
Author: Jonas Smedegaard <dr@jones.dk>
Last-Update: 2024-01-18
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -60,9 +60,11 @@
 linked-hash-map = "0.5"
 log = "0.4"
 memchr = "2"
+memmap2 = "0.9.4"
 mime = "0.3"
 num_cpus = "1.16"
 number_prefix = "0.4"
+object = "0.32"
 once_cell = "1.19"
 opendal = { version = "0.48.0", optional = true, default-features = false }
 openssl = { version = "0.10.64", optional = true }
@@ -106,7 +108,6 @@
 zstd = "0.13"
 
 # dist-server only
-memmap2 = "0.9.4"
 nix = { version = "0.28.0", optional = true, features = [
   "mount",
   "user",
@@ -114,7 +115,6 @@
   "signal",
   "process",
 ] }
-object = "0.32"
 rouille = { version = "3.6", optional = true, default-features = false, features = [
   "ssl",
 ] }
